pkg/auth/handler/forgotpwd/forgot_password.go

```diff
@@ -55,7 +55,7 @@ const ForgotPasswordRequestSchema = `
 	"$id": "#ForgotPasswordRequest",
 	"type": "object",
 	"properties": {
-		"email": { "type": "string", "format": "email", "minLength": 1 }
+		"email": { "type": "string", "format": "email" }
```

Comment:
It is better keep `minLength` because empty string is valid Go zero value.

Given a struct

```golang
type Params struct {
  OptionalEmail string `json:"email"`
}
```

It is difficult to write a schema for this struct while following Go convention to use string zero value to present absence of value.

Reply:
I think we're passing the raw JSON to the validator, so the Go convention should not matter?

Reply:
Oh, you're right. We have updated all validation code to validate against the raw JSON.